pennylane/templates/embeddings.py

```diff
@@ -39,107 +110,42 @@ def AngleEmbedding(features, wires, rotation='X'):
     The length of ``features`` has to be smaller or equal to the number of qubits. If there are fewer entries in
     ``features`` than rotations, the circuit does not apply the remaining rotation gates.
 
-    This embedding method can also be used to encode a binary sequence into a basis state. For example, to prepare
-    basis state :math:`|0,1,1,0\rangle`, choose ``rotation='X'`` and use the
-    feature vector :math:`[0, \pi/2, \pi/2, 0]`. Alternatively, one can use the :mod:`BasisEmbedding()` template.
-
     Args:
-        features (array): Input array of shape ``(N,)``, where N is the number of features
-            to embed. ``N`` must be smaller or equal to the total number of wires.
-        wires (Sequence[int]): sequence of qubit indices that the template acts on
+        features (array): Input array of shape ``(N,)``, where N is the number of input features to embed,
+            with :math:`N\leq n`
+        wires (Sequence[int] or int): int or sequence of qubit indices that the template acts on
 
     Keyword Args:
         rotation (str): Type of rotations used
```

Comment:
I'm terribly sorry - what exactly do you mean? The ``rotation`` argument? Shall I describe the allowed values of it rather than moving it to pos args?